internal/observability/event/options.go

```diff
@@ -113,7 +113,11 @@ func WithNow(now time.Time) Option {
 // WithFacility provides an Option to represent a 'facility' for a syslog sink.
 func WithFacility(facility string) Option {
 	return func(o *options) error {
-		o.withFacility = facility
+		facility = strings.TrimSpace(facility)
```

Comment:
Is this change in behavior needed? Should we add documentation about it?

Reply:
No it was defaulting higher up the stack, this just makes it more correct (and we updated the tests for it)